docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
+
+**Audio tagging**
+Audio tagging is a task to predict the tags of audio clips. This includes music tagging, acoustic scene classification, audio event classification, and so on.
+
+**Audio transcription**
+Audio transcription transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.
+
+**Automatic Speech Recognition (ASR)**
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
+A bounding box describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner.
+
+**Consolidated Receipt Dataset for Post-OCR Parsing (CORD)**
+COResets and Data Subset slection is an efficient and scalable library for making machine learning time, energy, cost, and compute efficient built on top of pytorch. It uses a subset of training data for model training and incorporates various state-of-the-art data selection strategies to select the proper representative data subset from massive datasets. 
+
+**Data annotation**
+Data annotation is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.
+
+**Data labeling**
+Data labeling is the process of identifying raw data (images, text files, videos, and so on) and adding one or more meaningful and informative labels to provide context so that a machine learning model can learn from it.
+
+**Data parsing**
+Data parsing is a method of converting data from one format to another format. For example, converting from HTML to JSON.
+
+**Dialog processing**
+Dialog processing or interactive document creation is the complementary method to batch processing and background processing. It is defined by the active user who creates all or part of the document in the system editor.
+
+**Docker**
+Docker is a software platform that allows you to build, test, and deploy applications quickly.
+
+**Document classification**
+Document classification is the act of labeling or tagging documents using categories, depending on their content.
+
+**Dataset**
+A dataset (or data set) is a collection of data comprised of individual items, or labeling tasks that you import into Label Studio.
+
+**Emotion recognition**
+Emotion recognition is the process of identifying human emotions.
+
+**Entity recognition**
+Entity recognition is a new technology that can enable investigators and document review managers to get to the pertinent data faster.
+
+**Human labeling**
+Human labeling enables you to label images in your training dataset at scale.
+
+**Image classification**
```

Comment:
Usually image classification is about the whole image classification, not at pixel-level

Reply:
Image classification is the process of categorizing and labeling images based on specific rules.